Ecosystem Partners/Marketplace Guides/getting-started-with-oracle-jdk-on-linux.md

```diff
@@ -0,0 +1,65 @@
+
+{{{
+  "title": "Installing Oracle JDK 1.8 on Linux",
+  "date": "12-1-2015",
+  "author": "CTL Ecosystem Team",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+![Java logo](http://www.oracle.com/ocom/groups/public/@otn/documents/digitalasset/1612441.gif)
+
```

Comment:
@andrewBrunette Please pull down and store the java logo in our images directory.